src/library/Uno.Themes.WinUI.Markup/Themes/FloatingActionButton.cs

```diff
@@ -0,0 +1,270 @@
+						public static ResourceValue<Brush> Focused => new("FabStateOverlayBackgroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabStateOverlayBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabStateOverlayBackgroundDisabled", true);
+					}
+				}
+
+				public static class Surface
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceForeground")]
+						public static ResourceValue<Brush> Default => new("FabSurfaceForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceForegroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("FabSurfaceForegroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FabSurfaceForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabSurfaceForegroundDisabled", true);
+					}
+
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceBackground")]
+						public static ResourceValue<Brush> Default => new("FabSurfaceBackground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceBackgroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("FabSurfaceBackgroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceBackgroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FabSurfaceBackgroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabSurfaceBackgroundDisabled", true);
+					}
+				}
+
+				public static class SurfaceStateOverlay
+				{
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceStateOverlayBackground")]
+						public static ResourceValue<Brush> Default => new("FabSurfaceStateOverlayBackground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceStateOverlayBackgroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("FabSurfaceStateOverlayBackgroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceStateOverlayBackgroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FabSurfaceStateOverlayBackgroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceStateOverlayBackgroundFocused")]
+						public static ResourceValue<Brush> Focused => new("FabSurfaceStateOverlayBackgroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceStateOverlayBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabSurfaceStateOverlayBackgroundDisabled", true);
+					}
+				}
```

Comment:
Should be inside of the `FloatingActionButton.Resources.Surface` class and renamed to `StateOverlay`